server/routes/pages.js

```diff
@@ -39,6 +39,50 @@ module.exports = function(server)
         event.emit('render', req, res, options);
     });
 
+    //match requested url with the news data url
+    function findPost(url){
```

Comment:
Please update this code to use the same indenting style as the rest of the project - https://en.wikipedia.org/wiki/Indentation_style#Allman_style

Reply:
i didnt know it had a name lol.